sphinx/screens/dashboard/dashboard/src/main/java/chat/sphinx/dashboard/ui/adapter/ChatListAdapter.kt

```diff
@@ -294,14 +301,24 @@ internal class ChatListAdapter(
                     } else {
                         includeDashboardChatHolderInitial.textViewInitials.text =
                             dashboardChat.chatName?.getInitials() ?: ""
-                        includeDashboardChatHolderInitial.textViewInitials
-                            .setBackgroundRandomColor(R.drawable.chat_initials_circle)
+
+                        onStopSupervisor.scope.launch(viewModel.mainImmediate) {
```

Comment:
same issue as noted previously regarding job/disposable cancellation on rebinding. Again, not sure where it went for the one on line 294-300, but.